pkgs/development/interpreters/php/fix-paths-php84.patch

```diff
@@ -13,15 +13,16 @@ index e46acf0928..ee8e5a88f8 100644
  
    AS_VAR_IF([GETTEXT_DIR],,
      [AC_MSG_ERROR([Cannot locate header file libintl.h])])
-diff -ru a/sapi/apache2handler/config.m4 b/sapi/apache2handler/config.m4
---- a/sapi/apache2handler/config.m4	2018-11-07 15:35:23.000000000 +0000
-+++ b/sapi/apache2handler/config.m4	2018-11-27 00:32:28.000000000 +0000
-@@ -66,7 +66,7 @@
-     AC_MSG_ERROR([Please note that Apache version >= 2.0.44 is required])
-   fi
+diff --git a/sapi/apache2handler/config.m4 b/sapi/apache2handler/config.m4
+index e335721f19..a5087e1320 100644
+--- a/sapi/apache2handler/config.m4
++++ b/sapi/apache2handler/config.m4
+@@ -68,7 +68,7 @@ if test "$PHP_APXS2" != "no"; then
+   AS_VERSION_COMPARE([$APACHE_VERSION], [2.4.0],
+     [AC_MSG_ERROR([Please note that Apache version >= 2.4 is required])])
  
--  APXS_LIBEXECDIR='$(INSTALL_ROOT)'`$APXS -q LIBEXECDIR`
+-  APXS_LIBEXECDIR='$(INSTALL_ROOT)'$($APXS -q LIBEXECDIR)
 +  APXS_LIBEXECDIR="$prefix/modules"
-   if test -z `$APXS -q SYSCONFDIR`; then
+   if test -z $($APXS -q SYSCONFDIR); then
```

Comment:
Is this an upstream issue ?

Reply:
It is one of long standing patch we have, we have a similar version for PHP < 8.4 https://github.com/NixOS/nixpkgs/blob/397a2540e62c3a4a9b87a1a5e5d4424c79e110bc/pkgs/development/interpreters/php/fix-paths-php7.patch

I updated it to resolve some conflicts without thinking about it too much but it is likely something could be upstreamed so we do not have to carry it.

Reply:
Fair enough, all good for me